packages/shared/src/api/postsApi.ts

```diff
@@ -47,10 +47,14 @@ export function channelPostAction(nest: ub.Nest, action: ub.PostAction) {
 export const sendPost = async (
   channelId: string,
   content: JSONContent,
-  author: string
+  author: string,
+  blocks?: ub.Block[]
```

Comment:
Should the signature just be `(channelId: string, authorId: string, content: ub.Story)`? Or why pass them in separately?

Reply:
Yeah, we should just create the story in the message input and send the whole thing up, just committed that now.